README.md

```diff
@@ -1,2 +1,41 @@
 # brpoplpush-redis_script
+
 Bring your own LUA scripts into redis
+
+## Installation
+
+Add this line to your application's Gemfile:
+
+```ruby
+gem 'brpoplpush-redis_script'
+```
+
+And then execute:
+
+    $ bundle
+
+Or install it yourself as:
+
+    $ gem install brpoplpush-redis_script
+
+## Usage
+
+TODO: Write usage instructions here
```

Comment:
We may add the documentation on a different pr if you want

Reply:
That is what I was thinking, I thought the default README.md from `bundler gem NAME` was a better start is all.